addons/test_discuss_full/tests/test_performance.py

```diff
@@ -101,7 +101,7 @@ def test_init_messaging(self):
         self.maxDiff = None
         self.env.flush_all()
         self.env.invalidate_all()
-        with self.assertQueryCount(emp=self._get_query_count()):  # ent 80
+        with self.assertQueryCount(emp=self._get_query_count()):  # ent 82
```

Comment:
move comment on the line with the value (or remove the comment if it's the same value)